sdcm/nemesis.py

```diff
@@ -513,6 +513,25 @@ def disrupt_switch_between_PasswordAuthenticator_and_SaslauthdAuthenticator_and_
         else:
             raise UnsupportedNemesis("SaslauthdAuthenticator can't work without saslauthd environment")
 
+    def disrupt_toggle_internode_compression(self):
+        def toggle_internode_compression(current_compression):
+            self.log.info(f"Current compression is {current_compression}")
+            values = ['dc', 'all', 'none']
+            values_to_toggle = list(filter(lambda value: value != current_compression, values))
+            return random.choice(values_to_toggle)
+
+        self._set_current_disruption(f'toggle_internode_compression {self.target_node}')
+        key = 'internode_compression'
+        with self.target_node.remote_scylla_yaml() as scylla_yaml:
+            current = scylla_yaml.get(key, 'no_internode_compression_key')
```

Comment:
what is `no_internode_compression_key`? can't find it in the Scylla YAML

Reply:
@bentsi 'no_internode_compression_key' is a dummy value, to specify key does not exist, I'll rename to 'dummy_value..'  to avoid confusing.